### ISSUE TITLE
Accessibility feature of none and conformance

### ISSUE THREAD

**Issue by mattgarrish** [mattgarrish/epub-revision#2537]
It would probably help to note in the accessibility techniques document that although the schema.org vocabulary includes the value "none", setting it does not meet the requirement to identify accessibility features.

I know we allow "unknown" hazards to pass for political reasons, but I can't think of why setting "none" would be acceptable for features.

**Comment by mattgarrish** [comment]
Closing as this was fixed by #2547
